websites/A/AnizmTV/dist/metadata.json

```diff
@@ -13,8 +13,40 @@
 		"vi_VN": "AnizmTV là trang web giúp bạn xem anime phụ đề mới nhất với chất lượng cao."
 	},
 	"url": [
+		"www.anizm.info",
+		"anizm.info",
 		"www.anizm.net",
-		"anizm.net"
+		"anizm.net",
+		"www.anizm.tv",
+		"anizm.tv",
+		"www.puffytr.com",
+		"puffytr.com",
+		"www.anizle.com",
+		"anizle.com",
+		"www.anizm.pro",
+		"anizm.pro",
+		"www.turkanime.info",
+		"turkanime.info",
+		"www.turkanime.pro",
+		"turkanime.pro",
+		"www.turkanime.shop",
+		"turkanime.shop",
+		"www.turkanime.xyz",
+		"turkanime.xyz",
+		"www.turkanime.store",
+		"turkanime.store",
+		"www.anizm.store",
+		"anizm.store",
+		"www.tranimeizle.org",
+		"tranimeizle.org",
+		"www.animetr.fun",
+		"animetr.fun",
+		"www.tranimeizle.xyz",
+		"tranimeizle.xyz",
+		"www.turkanime.fun",
+		"turkanime.fun",
+		"www.turkceanimeizle.org",
+		"turkceanimeizle.org"
```

Comment:
A lot of these URLs are useless, most of the ones starting in www. do not exist if not all

Reply:
no they do not work. If you enter one you are redirected to the one without www

Reply:
I remove www links.